### ISSUE TITLE
Secondary initiator name is not used in multipath ProutPreempt tests

### ISSUE THREAD

**Issue by sitsofe** [sahlberg/libiscsi#187]
When running the ProutPreempt tests normally here are the results:

```
Suite: ProutPreempt
  Test: RemoveRegistration ...
    Test Persistent Reserve IN PREEMPT works.
    Send PROUT/REGISTER_AND_IGNORE to register init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test
    Send PROUT/CLEAR to clear all registrations and any PR reservation
    Send PROUT/REGISTER_AND_IGNORE to register init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test
    cloning sd1 for second session
    Send PROUT/REGISTER_AND_IGNORE to register init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test-2
    Send PRIN/READ_KEYS
    Send PROUT/PREEMPT to preempt reservation and/or registration
    Got UA for TUR
    TUR good after 1 retries
    TUR good after 0 retries
    Send PRIN/READ_KEYS
    Send PROUT/REGISTER to unregister init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test-2
passed
```
(You might need to scroll right to see the full initiators being used)
When using multipath here are the results:
```
Suite: ProutPreempt
  Test: RemoveRegistration ...
    Test Persistent Reserve IN PREEMPT works.
    Send PROUT/REGISTER_AND_IGNORE to register init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test
    Send PROUT/CLEAR to clear all registrations and any PR reservation
    Send PROUT/REGISTER_AND_IGNORE to register init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test
    using multipath dev for second session
    Send PROUT/REGISTER_AND_IGNORE to register init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test
    Send PRIN/READ_KEYS
    Send PROUT/PREEMPT to preempt reservation and/or registration
    [FAILED] PROUT command: failed with sense. RESERVATION CONFLICT
    TUR good after 0 retries
    TUR good after 0 retries
    Send PRIN/READ_KEYS
    Send PROUT/REGISTER to unregister init=iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test
FAILED
    1. test_prout_preempt.c:81  - CU_ASSERT_EQUAL(rk->num_keys,2)
    2. test_prout_preempt.c:91  - CU_ASSERT_EQUAL(ret,0)
    3. test_prout_preempt.c:104  - CU_ASSERT_EQUAL(rk->prgeneration,old_gen + 1)
```

In the multipath case the iqn.2007-10.com.github:sahlberg:libiscsi:iscsi-test-2 initiator name is never used...

**Comment by sitsofe** [comment]
This should be addressed by the pull request #203 .

**Comment by sahlberg** [comment]
Merged.
